<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterCategoryForm</class>
 <widget class="QWidget" name="MasterCategoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="SelectedRemark">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Remark</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Saved category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdateButton">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>8</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset resource="ServiceLib.qrc">
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="SelectedFolderName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>FolderName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
   <property name="icon">
    <iconset resource="ServiceLib.qrc">
     <normaloff>:/Qtres/Change.png</normaloff>:/Qtres/Change.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ServiceLib.qrc"/>
 </resources>
 <connections/>
</ui>
